tools/check_vendored_modules.py

```diff
@@ -14,7 +14,38 @@
 NAPARI_FOLDER = "napari"
 
 
-def check_vendored_module(org : str, reponame : str, tag : str) -> str:
+def _clone(org, reponame, tag):
+    repo_path = TOOLS_PATH / reponame
+    if repo_path.is_dir():
+        shutil.rmtree(repo_path)
+
+    check_output(
+        [
+            "git",
+            "clone",
+            '--depth',
+            '1',
+            '--branch',
+            tag,
```

Comment:
That will not work with an exact hash, but is faster when giving a tag or a branch as it clones only what is necessary.